MiniApp/Classes/RealMiniApp.swift

```diff
@@ -48,6 +48,13 @@ internal class RealMiniApp {
                 }
                 self.downloadMiniApp(appInfo: appInfo, completionHandler: completionHandler, messageInterface: messageInterface)
             case .failure(let error):
+                let createError = error as NSError
+                if Constants.offlineErrorCodeList.contains(createError.code) {
```

Comment:
I will move the `offlineErrorCodeList` close to the class but I don't think we can move handleDownloadError to this class. 
And the reason is when **createMiniApp** is called in **RealMiniApp**, it will check the platform for the latest mini-app info, if the request is failed(offline) then we load the cached mini-app, if the request is a success(online) then we continue downloading/cache the version(if it is already downloaded).
So we need to take care of returning the cached version of mini-app in two places esp in offline mode,
1. **RealMiniApp** - while fetching the Mini app info - request failed - serve the cached mini app
2. **MiniAppDownloader** - while downloading in progress - download failed - serve the cached mini app

We can combine these into one, but we need to move the `MiniAppInfoFetcher` implementation into `MiniAppDownloader`